### PR TITLE
feat: added landing page

### PR DESCRIPTION
**Pull request by pradipgharge**
created the basic version of landing page

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    width: 100%;
+}
+
+.btn-icon-img{
+    display: flex;
+    justify-content: center;
+}
+
+.navbar-right{
+    padding:0rem 2rem;
```
Comment (sri-rishi) [inline style.css:64]: use 0  instead of 0rem, it will work

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Great work, just some suggestion I have commented them

File: style.css (hunk 1 above)
Comment (pradipgharge) [reply style.css:64]: ok thanks for the review.

**Review by parvez-rashid — APPROVED** [review]
I like the minimalist style of the landing page. Code looks good to me. All the best!

**Review by Sahil25061999 — APPROVED** [review]
suggestion: the flex property is repeating alot, so you just separate out the property and give all the classes togther. 
UI looks great.

**Comment by anasm11** [comment]
Color of text on top of image could be changed to make more readable. Rest all looks good.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    align-items: center;
+    margin-top:2rem;
+    padding: 1rem 0;
+    font-size: 1.1rem;
+    color: var(--tertiary-text-color);
+    background-color: var(--solid-black-color);
+}
+
+
+
```
[the remark this answers is not in the document]
Comment (pradipgharge) [reply style.css:179]: Done

**Comment by pradipgharge** [comment]
> suggestion: the flex property is repeating alot, so you just separate out the property and give all the classes togther. UI looks great.

Yep right, but the other properties coming with flex are varying so I kept it this way.

**Comment by pradipgharge** [comment]
> Color of text on top of image could be changed to make more readable. Rest all looks good.

Tried a few colors, black was best fit also its the primary color for the site so used it. Thanks for the suggestion though

**Review by AshutoshM29 — APPROVED** [review]
UI looks good, you can merge it!!

**Review by sri-rishi — APPROVED** [review]
Changes looks good, can be merged

**Pull request merged**